.github/workflows/playwright-cloudflare-open-next.yml

```diff
@@ -51,6 +51,12 @@ jobs:
       - name: Install packages
         run: pnpm install --frozen-lockfile
 
+      - name: Downgrade Next.js
+        # Open-next does not yet support next@15.4, so temporarily in order to have
+        # this workflow working we force install the latest Next.js 15.3 version instead
+        run: pnpm i next@15.3.2
+        working-directory: apps/site
+
```

Comment:
Isn't this still a known vulnerable version?

Reply:
Basically the whole point here is to make sure that the open-next build doesn't break for unrelated reasons (as it happened for `fs`'s `glob` in this case for example). Since that could make it much harder to get everything back on track once open-next does support Next.js 13.4